db.py

```diff
@@ -0,0 +1,277 @@
+        cur.close()
+        return False
+    else:
+        cur.execute("DELETE FROM statistic WHERE statistic.question_id = (%s)", (question_id,))
+        cur.execute("DELETE FROM choice WHERE choice.question_id = (%s)", (question_id,))
+        cur.execute("DELETE FROM question WHERE question.id = (%s)", (question_id,))
+        conn.commit()
+        cur.close()
+        return True
+
+
+@decorator_add_connection
+def get_question(user_id, conn=None):  # Получаем уникальный вопрос
+    question_id = find_not_answered(user_id)
+    if not question_id:
+        return False  # Если на все ответили или вопросов нет возвращаем False
+    else:
+        cur = conn.cursor()
+        cur.execute("""SELECT question.question_text, choice.choice_text FROM question
+    JOIN choice ON question.id = choice.question_id and choice.question_id = (%s)""", (question_id,))
+        select = cur.fetchall()
+        select_list = [list(b) for b in select]
+        cur.close()
+        answer = f"{select_list[0][0]} \n"
+        number = 0
+        for i in select_list:
+            answer += f"{number + 1}: {i[1]}\n"
+            number += 1
+        return [answer, question_id]
+
+
+@decorator_add_connection
+def answer_question(tg_user_id, question_id, choice_id, conn=None):  # Отвечаем на вопрос
+    conn.autocommit = False
+    cur = conn.cursor()
+    number = find_empty_id('statistic')
+    cur.execute("""INSERT INTO statistic VALUES ((%s),(%s),(%s),(%s))""", (number, tg_user_id, question_id, choice_id))
+    conn.commit()
+    cur.execute("""SELECT votes FROM choice
+    WHERE id = (%s) and question_id = (%s)""", (choice_id, question_id,))
+    votes_number = cur.fetchall()
+    votes_number_list = []
+    for _ in votes_number:
+        for item in _:
+            votes_number_list.append(item)
+    votes_number = votes_number_list[0] + 1
+    cur.execute("""UPDATE choice
+SET votes = (%s)
+WHERE id = (%s) and question_id = (%s)""", (votes_number, choice_id, question_id))
+    conn.commit()
+    cur.close()
+
+
+@decorator_add_connection
+def get_all_statistic(conn=None):  # Получаем общую статистику
+    cur = conn.cursor()
+    cur.execute("SELECT * FROM public.question ORDER BY id ASC ")
+    questions = cur.fetchall()
+    if not questions:
+        cur.close()
+        conn.close()
+        return 'Повелитель, вопросов еще нет. Создайте их, воспользовавшись соответствующей функцией в меню'
+    else:
+        questions_list = [list(b) for b in questions]
+
+        answer = ''
+        for i in questions_list:
+            answer += f"\nВопрос {i[0]}: {i[1]} \n"
+            answer += f"Ответы: \n"
+            cur.execute("SELECT choice_text, votes FROM choice WHERE question_id = (%s)"
+                        "ORDER BY id ASC", (i[0],))
+            choices = cur.fetchall()
+            choices_list = [list(b) for b in choices]
+            for f in choices_list:
+                answer += f"- {f[0]} -- {f[1]} \n"
```

Comment:
https://github.com/thehighestmath/umschool-final-task/blob/master/review_comments.md#2-%D0%BA%D0%BE%D0%BD%D0%BA%D0%B0%D1%82%D0%B5%D0%BD%D0%B0%D1%86%D0%B8%D1%8F-%D1%81%D1%82%D1%80%D0%BE%D0%BA-%D0%B2-%D1%86%D0%B8%D0%BA%D0%BB%D0%B5---%D1%8D%D1%82%D0%BE-%D0%B7%D0%BB%D0%BE